conjure-java-jersey-server/src/main/java/com/palantir/conjure/java/server/jersey/InternalErrorExceptionListener.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * (c) Copyright 2020 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.conjure.java.server.jersey;
+
+import com.palantir.conjure.java.api.errors.RemoteException;
+import com.palantir.conjure.java.api.errors.ServiceException;
+import com.palantir.conjure.java.api.errors.UnknownRemoteException;
+import com.palantir.logsafe.SafeArg;
+import com.palantir.logsafe.exceptions.SafeIllegalStateException;
+import com.palantir.tritium.metrics.registry.SharedTaggedMetricRegistries;
+import com.palantir.tritium.metrics.registry.TaggedMetricRegistry;
+import feign.RetryableException;
+import java.io.IOException;
+import java.util.Optional;
+import java.util.function.Consumer;
+
+final class InternalErrorExceptionListener implements Consumer<Throwable> {
+    private final JerseyServerMetrics metrics;
+
+    InternalErrorExceptionListener(TaggedMetricRegistry registry) {
+        this.metrics = JerseyServerMetrics.of(registry);
+    }
+
+    /** Records information to the singleton TaggedMetricRegistry. */
+    static Consumer<Throwable> createDefault() {
+        return new InternalErrorExceptionListener(SharedTaggedMetricRegistries.getSingleton());
+    }
+
+    @Override
+    public void accept(Throwable exception) {
+        Optional<ErrorCause> errorCause = classifyTypeOfInternalError(exception);
+        if (errorCause.isPresent()) {
+            ErrorCause cause = errorCause.get();
+            metrics.internalerrorAll(cause.toString()).mark();
+        }
+    }
```

Comment:
@carterkozak this class is just preserving the behaviour that was already working in c-j-r, but I consider this literally as a stopgap thing because I want to _immediately_ swap it out for WC's implementation.

Reply:
Ack, can we take advantage of Jerseys dependency injection framework here?

Reply:
Chatted over VTC and figured that there aren't actually _that_ many prod usages of ConjureJerseyFeature.INSTANCE (since 99% of servers get it for free in WC). I think this means it's reasonable for any stragglers to just use the builder.